pinot-core/src/main/java/org/apache/pinot/core/data/readers/AvroRecordReader.java

```diff
@@ -81,8 +82,9 @@ public GenericRow next(GenericRow reuse)
       String fieldName = fieldSpec.getName();
       Object value = _reusableAvroRecord.get(fieldName);
       // Allow default value for non-time columns
+      //Why special treatment for Time Column?
```

Comment:
Open an issue so someone can respond to why was time column given special treatment. Hopefully, they can also add comment in the code explaining the same.

Reply:
I think I got the answer in NullValueTransformer PR.